src/api/soap/schema.ts

```diff
@@ -489,7 +489,7 @@ export const ApiSchemas = {
   UndeleteResult: {
     type: 'UndeleteResult',
     props: {
-      errors: ['Error'],
+      errors: ['?', 'Error'],
```

Comment:
did you update this manually or via an updated WSDL? See the comment at the top of this file

Reply:
Manually, I just read the top of the file...
I'll fix it